packages/common/src/index.ts

```diff
@@ -1076,7 +1078,7 @@ export default class Common extends EventEmitter {
    *
    * Note: This value can update along a hardfork.
    */
-  consensusType(): string | ConsensusType {
+  consensusType(): string | ConsensusType | undefined {
```

Comment:
This should always have some decent return value, so adding `undefined` is somewhat a step in the wrong direction (and would also be somewhat of a breaking change, at least on the TypeScript level).

I added this in the `Chain` interface as optional for some backwards compatibility reasons, see also the associated TODO "// TODO: make mandatory in next breaking release".

We should nevertheless just use the `!` operator here (below) instead of `?`. It's not 100% clean but we have consensus all defined in our own chain definitions and it would very much be the greater annoyance if we introduce breaking changes here which somewhat only theoretically apply.

Reply:
Makes sense, changing for `!`